dotCMS/src/main/java/com/dotcms/publisher/util/DependencyManager.java

```diff
@@ -88,7 +89,7 @@
 	private DependencySet folders;
 	private DependencySet htmlPages;
 	private DependencySet templates;
-	private DependencySet structures;
+	private DependencySet contentTypes;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Private field 'contentTypes' could be made final; it is only initialized in the declaration or constructor.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5458951)